### PR TITLE
feat: add start api doc page

### PR DESCRIPTION
**Pull request by metal-messiah**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context
* What problems does this PR solve?
    - This PR adds a documentation page for a new API in the browser agent --> `.start()`
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
    - https://github.com/newrelic/newrelic-browser-agent/releases/tag/v1.239.0
* If your issue relates to an existing GitHub issue, please link to it.
    - https://issues.newrelic.com/browse/NR-154372

### CODE REVIEW

File: src/content/docs/browser/new-relic-browser/browser-apis/start.mdx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+tags:
+  - Browser
+  - Browser monitoring
+  - Browser Agent
+metaDescription: A method used to start agent features when running in a deferred state
+redirects:
+---
+
+<Callout variant="important">
+This API will work for any browser edition (Browser Lite, Pro, or Pro+SPA). The configuration required to use this API is not currently connected to the larger deployment system within New Relic.  As such, this feature is only supported via Copy/Paste or NPM until further changes are made.
```
Comment (patrickhousley) [inline src/content/docs/browser/new-relic-browser/browser-apis/start.mdx:14]:
```suggestion
This API will work for any browser edition (Browser Lite, Pro, or Pro+SPA). The configuration required to use this API is not currently connected to the larger deployment system within New Relic.  As such, will only have an effect in Copy/Paste or NPM installations until further changes are made.
```

File: src/content/docs/browser/new-relic-browser/browser-apis/start.mdx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+</Callout>  
+
+
+
+## Syntax
+
+```js
+newrelic.start(value: string[]|null)
```
Comment (patrickhousley) [inline src/content/docs/browser/new-relic-browser/browser-apis/start.mdx:22]:
```suggestion
newrelic.start(value: string[]|undefined)
```

Comment (metal-messiah) [reply src/content/docs/browser/new-relic-browser/browser-apis/start.mdx:22]: nice catch :ty:

File: src/content/docs/browser/new-relic-browser/browser-apis/start.mdx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `value`
+
+        _string[]_ OR _null_
```
Comment (metal-messiah) [inline src/content/docs/browser/new-relic-browser/browser-apis/start.mdx:60]:
```suggestion
        _string[]_ OR _undefined_
```

File: src/content/docs/browser/new-relic-browser/browser-apis/start.mdx [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+tags:
+  - Browser
+  - Browser monitoring
+  - Browser Agent
+metaDescription: A method used to start agent features when running in a deferred state
+redirects:
+---
+
+<Callout variant="important">
+This API will work for any browser edition (Browser Lite, Pro, or Pro+SPA). The configuration required to use this API is not currently connected to the larger deployment system within New Relic.  As such, will only have an effect in Copy/Paste or NPM installations until further changes are made.
```
[the remark this answers is not in the document]
Comment (metal-messiah) [reply src/content/docs/browser/new-relic-browser/browser-apis/start.mdx:14]: I don't think that currently exists on the docs site

**Review by akristen — APPROVED** [review]
Looks good!

**Pull request merged**